endpointmanager/README.md

```diff
@@ -167,6 +210,17 @@ go run main.go
 
 The commands below assume that you are starting in the root directory of the Lantern backend project.
```

Comment:
I think `CHPLupdatecheck` is the only command missing in this section? I think it's also a good idea to move these to alphabetical order so they mirror the order that they're in for the code